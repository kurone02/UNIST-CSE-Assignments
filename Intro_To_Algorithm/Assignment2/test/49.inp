2735
1796686619
1352858002
330930790
2036279405
1243098647
226663305
1010885867
1993877421
1098514442
1608405313
815952882
265294938
185171350
1531547465
420932849
1731611092
2140272261
85219304
540673322
526570179
2033853301
63917615
1483652117
1579422214
1134715444
677257984
1487210144
1157545704
1498954379
2088681338
301485101
1591087344
710258916
19508132
483712525
233480445
756807715
1170677750
1440486715
800292620
1790619467
198922945
962990024
1533833323
374506118
1973507654
1161874888
977864100
245991850
397468621
1372447788
1998177432
53916176
144581341
1043732714
860888814
2107421007
2097843645
355219033
1189590469
228562766
2003254011
1512234433
377259114
1631379647
373826941
2139765043
734155935
1140582401
1171804061
1887040089
1504160555
251807887
1461315612
75875439
325459133
243513521
2033019681
992732232
648779918
220766747
558225901
409937427
729968522
790725385
1684939487
1708757838
1550206728
1142843708
1394631872
2013272987
258990349
1322534225
1829706413
1635997327
1110167231
1431954936
1710633259
1198613462
1220662775
949970500
424444929
407458776
2127026582
1031565596
146256796
1560713173
1781774544
439561722
1949365592
1322917309
331605509
1734254007
1994933197
1659940592
317842060
1700545780
1287167253
1693964869
161092769
1225962328
122077140
1089644187
855878124
71221516
65567723
2059736647
1619281419
1278079854
2015099816
1728358583
583252110
1192245865
1536366298
1191789364
1192158178
1529052519
241018887
343835612
55108687
1479799968
1529317646
1373535818
2002527737
269482189
1529114794
132698845
1860257193
695827895
800836441
1556719255
1051932100
1651563307
269152415
1028710332
1167603337
419928112
752338487
844557492
1518668122
137823114
19756703
1145202057
1744382377
1969356039
1323308648
288464582
1913769595
1425933816
401305311
810186569
1321037004
243513589
1559786927
542935883
265251112
837834289
1861541078
1358716091
1043343189
1253459427
206840362
1824881060
601296684
1149994915
2087887565
2025564620
1375155479
1002854521
108709291
684943170
301188335
582620685
1587643485
1575166272
147136419
315871431
219592572
1611730528
840508682
588321347
1328586013
865401060
1951018134
427644628
2082280191
1671453228
1481733839
19525157
2057707171
1995870759
1404321241
9923066
1155702428
751864195
217824252
1731453457
833451893
80854706
1650459008
1430404936
1843793331
1129312596
1854352065
2101648114
99501310
1456043841
1220153611
1001436551
2124604945
1279367306
1906501634
570578666
313935765
1188671867
1355791032
832886805
11166687
26286338
1946038552
194551419
373227886
1563813524
1779013251
1758755275
404828160
643284514
1467300252
792430249
1632994540
1592997847
348418143
1869907446
1511232094
1077329040
1597714611
920882198
666295195
1559619939
870024047
357271468
1318790201
1754955803
975416056
257348566
354925309
1321091169
1884630126
2044268885
1544213174
258141905
1583561956
618420136
2354677
1207299382
1440537870
623151406
703727476
1465686566
1929028972
1100242409
1299523599
1471315002
300248370
1746200495
1066229558
1320959968
534854833
392848549
654152751
1111615440
989761186
432551290
813118429
1837522761
1908096139
1508803409
664914078
1817864761
929053095
1923879550
1514746722
1642036271
1244241766
2136272448
1642695160
1450574411
562283318
1557587785
360956308
402223265
1011521011
1657271447
1171179848
803742443
1953896512
1986114088
675554356
997238171
496283277
2032511345
1649989661
103158709
1280061439
1672085989
702408477
1613945254
109414496
498953135
2075811036
121092336
354853467
755069322
1236120836
1642474800
1156106759
1643600753
932985133
688253601
753928651
365502525
1209972536
691913792
1158922078
1735961744
1749229964
557568050
506655540
2047262527
1531752707
95877770
1065686672
1701414051
622774419
466237722
78774358
1341188184
295886154
2072868074
63157523
1865310200
1323134108
2050397211
592187702
204612725
552802224
828122694
1878454104
955334320
1296728022
1999527228
1558471198
355837285
1473539664
105531770
1790827471
1597912597
617340578
181733365
2118645620
1388454670
1156273513
1891209929
1737064543
1506715499
331197064
609748496
1715794667
334407749
1090304903
1105772647
1628743447
1469977808
1414418196
571230711
2076752264
1937618454
1575331499
623478841
1320002474
716956226
103594676
299155227
1128339966
492708302
1577961288
596956850
1934623378
495211344
1545909249
1869924600
655309137
696127072
9855823
718313109
1001630276
1806952233
1605840719
140115403
1514734776
1555619951
1982946998
220462980
1940398402
850010859
149719885
871419417
1897152134
743982435
1987545686
1397142262
1490357310
1211720594
145456998
599545236
1619369276
1056177646
502556330
647172432
506931275
151259596
889860317
1990592711
1320400068
1119462125
705010117
1566974317
1298408631
2128798804
537591204
1985514878
1531622548
1005987730
425032301
340125283
1316246611
1926085659
509875090
88165650
538944844
1850000998
1846940382
1766822645
1590017194
1778776529
1738218976
929929315
259312785
662680741
208063529
692764643
1334503904
1845085760
571108036
1911029466
760927783
1936561689
797395192
209499805
1611816029
53396925
1941130462
2031352751
2009096886
497955897
1437712317
147712648
1376903665
106442847
338370340
900571860
911620131
354580036
41753694
1680708575
140033233
2083272214
1430581473
583428619
1218042285
1736859780
747284869
1353424434
1471341024
2104878327
398986917
415237417
403361058
33977698
1564711916
1408278488
550922074
1607921145
200077902
550786021
232337629
962667228
951651592
384644770
492817048
926015229
516529486
12684124
1865337435
118802988
1795081073
1566653793
1717607417
312582713
654431446
647598218
1906537756
1177553671
1059190685
2089142179
549906983
1453750045
1891053651
848364551
973703904
1233455940
1878398672
400691027
168035160
449957448
587229470
1263079720
1023439995
420969678
1642155407
2114566949
921182758
222132862
1928174366
235706030
586582169
2076806554
1309691048
534632637
2043328040
1426698574
662947865
1088949543
1712194889
1927799848
1740714116
731503508
149031353
1845331225
1369200152
2098422323
1919554309
1448600146
1522211158
1316494469
1078192615
2040656166
743927644
1039158709
2133954640
1335590939
952353864
153833789
1760335217
1205476037
2134909822
769200111
2045592070
1537003675
262138614
480083034
1159039502
2050696221
1783930712
2090850611
243523642
617292362
696141305
2107173652
2008351582
1690946693
1561052741
431803649
1668032146
192217691
1306069244
1373969247
1699427121
328819151
510239902
1854552267
1201086140
1283080914
1298820759
1773152843
1967760439
2091556208
583237611
672734330
1871426514
1246322123
728890415
537323878
1205029814
1653600749
269674740
1533125416
288540364
624701096
1169787865
140536381
1201164564
2048699261
1190057572
450155459
476149854
2038771106
59011158
1724738828
1857624507
214042927
707519102
1733471219
1411479762
1509705079
659042794
270519393
30111904
717458778
465424096
1955572673
1863390699
1539528501
453014670
170348513
1487538630
1439864717
542717547
1936137527
1023184691
2055328910
1139036100
766876901
825724111
1497977630
53390928
2021643333
1124093091
1663081121
1065106080
145806880
746974339
88402181
1926157451
131015761
2108618726
213693080
718445006
495147258
178218688
1573293208
2062774439
1572828795
1796083804
2147346979
362660871
49982222
670733363
1307531889
1201651241
61546054
314221825
1656452071
1679620926
1876593403
1761345604
814549949
23577999
1531052572
993521793
154184413
469770887
342329371
1135485957
1905550186
535480986
318007314
635844719
1034973952
836393158
1806311813
1761141240
1971393356
1145195978
814040699
2048605309
1591617418
967293991
1624323102
133341928
1001820869
2141504733
1953624250
1533573459
1284622809
231473647
1914907697
175554170
412205306
235728275
1427659281
2115792343
464852435
1088781313
971039366
1586772022
964025338
812819067
831841879
1646661363
573780392
1744979114
2005767595
373357915
1461705040
639224144
1774034907
1634812363
992853405
39035749
60071430
247898349
1975645324
1818223989
527127525
799511216
1374535093
1551698729
1728019418
45412309
23267717
1483233120
2071050406
1946730651
1991386262
153308146
1968108894
1986086942
1340941786
1586424891
296070840
1973830735
885443220
1072037087
896305939
1794131916
1433865414
2063825138
1631035046
711271965
465309101
662216424
896220292
647637552
1839997383
1554529930
132556040
628870143
1925216584
874442560
96492133
323304624
1411607374
1769310882
133283311
1289770198
1941152113
110050076
1501983282
1210440428
189162687
1589513368
1145686665
119455050
670266157
1406026739
172114681
2039707385
451130888
1925727955
216368648
289795268
1944866381
1599191479
1468786674
747081738
823595182
1016426393
702108052
1478449709
941340160
1596929720
1344682772
42141209
477572083
872474963
297575133
736804783
1040082630
2017740000
575527569
128613627
941655905
805183314
187768917
1637412737
20532313
270610496
480175196
2111995870
738830118
704524160
935395496
505877512
490742540
1294020468
1182258171
1545984038
1867814369
560795235
1925293512
1241958016
1020633407
1847807484
1657317986
559276110
2082160841
2060879064
2000599641
2125083788
1727539481
300701151
2022006193
1442948267
499801368
362102707
980736682
755408449
1166813996
301968168
935491243
1995962916
1168988592
1844912940
1233832697
922406724
294175483
614287622
1897166194
534559878
397441247
1244800555
269019650
562390406
2077691367
1534045871
790026008
1207781719
1708740435
1669590731
352381276
1026273201
1486251675
534100267
63133680
1536456034
1621554616
1807935492
1999301233
1883942006
836221584
784281586
631534538
1598665421
2119925507
1316900013
1416375588
1832697369
1186367993
2004641491
715952024
1481526770
105238781
419052135
1905219329
1399208980
513731949
772855772
338889937
1718334088
224651037
854405025
1498650808
1703495603
968287620
898189122
709925071
2077466294
857658344
703146099
1682454196
1591025384
923100320
671391783
1058054408
539106153
1881745572
1662638469
1036418730
970619203
60958981
1689097812
1588113729
61334796
618562362
1974197434
1839490753
2066409025
1944998208
2043849062
968618302
2023903957
1617616250
293368023
529241804
87859731
960676690
372321878
1756750151
1127526421
819933744
1732892604
1601471927
1925368116
410576577
529333325
1988833330
2117236848
29976023
2033786493
916577146
892747739
453681572
1935858848
1780662158
441341209
364845202
587319698
673298346
732877
1383163020
703818184
592709991
1454009088
1883284882
1032545836
696402251
1264912026
879781713
1435555857
246150244
2141850235
875807255
978713316
1494118741
1125248147
259681051
862310935
630237364
2141750498
2063446706
1509884631
475855926
78986508
661697505
239004341
1428431255
1525760820
1098472052
1156329743
1322594790
1358222864
2008262008
1804124211
251916430
1065967746
1521342868
1160092676
1960284214
1959724180
435419818
1916664997
1828066418
2022239498
1738498276
819117876
1331910125
2129321017
1933522226
667505607
294627216
1953050106
1085931088
1474471590
1270594688
1197319017
1949170751
1699164380
104098573
1509726731
577405514
1726227013
54729387
1405514650
1420833002
207945470
203620787
353391418
518389439
34966634
1985401066
420866837
460065614
899604711
1602344757
219178194
264085475
248974253
874718262
179922607
650371953
2110382588
377144700
1992161021
1784708165
943298466
1745082743
1011224596
1313539541
1688009135
535388136
446162380
1192587087
1495977021
1396476580
1090415161
624209673
1016056590
1595621952
77508541
315990087
348584711
264951241
259519598
973630697
1757031343
650447769
215301344
2030905631
2027206969
769598752
457832047
64269183
1439999930
66709154
1452654894
1649837710
631390663
173512628
320914218
120055313
941436061
217642480
381562221
381219317
1201523235
1582987651
143657212
1744647862
1773284406
378500218
279107452
114807857
508352538
2033195611
721352703
153821222
1235875296
1364566473
2034570822
1271325335
1348281874
797585188
1944667510
1133190083
882223475
2057984021
471856841
814240756
1237479261
1962431858
1314746868
363864803
1291390712
1704160002
421796831
526205606
1206295886
951032857
878645376
1955282955
532366241
1736763340
1314524390
2022089186
90619221
792788070
1399537610
1924981140
849676663
312542025
2120808889
544520405
972741236
1923173909
235692871
1744709978
1283092543
1003292911
1214412413
1105680390
1314876162
1968822873
1239996487
1508334883
1790825603
420163514
816475488
1522562499
1539303196
192851621
147647998
1767712604
329705784
1903252954
371484789
1084372719
1555445834
431417448
1425576713
365807244
495643439
614596796
132282011
993192906
1707272854
789059519
503214985
315740805
1008436020
1350045434
1404324197
1141194035
1434771534
2122373389
1650478171
1582351424
438194524
1478004303
1337016652
1200550148
412568257
1668601556
929500542
206000416
771523558
1787628461
1247482141
1819303058
91731075
2002303129
1352782040
1906050344
944082081
1543124927
2108513487
455555549
1298065777
1028798413
1638714413
219710120
1503718786
1548595866
205995439
1538301985
1860191513
1556278075
649041159
1958108295
449573572
868756806
2083321608
1139718308
100609589
211991821
845864081
1133631124
1485681857
1233650883
198896400
1642665220
775100347
1004782535
1950507909
873671956
479922225
1199923078
198871141
1943456994
1719572553
1433324888
350076772
1468083609
1825009976
579258170
495372963
687796036
1992837158
1999934189
153572841
1912291838
1548339068
2076785000
1397388905
1278636605
1569827959
671553273
44741022
556620784
120354659
503975033
65611280
1403382706
1192325328
1502652203
1443075778
171685157
908487714
100092220
756246927
2040957011
810350321
1002715907
906985177
988896573
1579079487
378399623
1045114802
1322490496
904414455
32745243
564776711
392002685
669920341
146189467
1879212169
67793236
472420259
894576862
1883833446
2042001732
713166177
1403334727
1372018536
1084876170
1404364287
27233713
306478317
1848637417
1196741715
358245541
1426407759
130133972
1118883223
768418923
533818762
1204046178
41131767
1302982889
934963663
1489311859
325540388
1503839523
382401475
1369964197
1464220723
1997508658
855295682
1334715822
1843268469
577350707
1054695909
1850300143
60418502
1623874890
110479775
1991783388
1689209661
2071327316
502644452
1940812291
666861966
383960337
43293723
2110228799
1096253919
455729567
1984442028
945742236
958035584
856218012
1437125477
123812964
439350047
444477597
1811654869
1913398136
891755801
412239785
274719052
605958618
1864788722
1447524192
1944429947
601395575
673144433
875705382
857243407
1473352639
813491756
1568271043
883361031
831737266
1263560854
892760465
1114721594
1156543254
679531484
1889557678
204943208
1105607724
245004458
482010292
1443993549
1067820720
1250609636
1173958135
2031637703
141761203
1155539122
783108886
2111326141
1678210597
722157783
575004056
1975166318
419327037
293333346
47626149
7306624
1666341491
1761493099
205267573
1376214435
1202289491
502613980
1633441634
314433634
1191072195
1703745196
885021095
1970501219
1829793082
158873513
700552861
1785629038
1714985282
390677949
2107683316
894444334
1846107635
1325468097
762968554
492076298
2144295828
225103968
1404938928
179530533
373912216
915411771
854629975
1847055793
48191418
1056679979
1726132352
255513081
218822957
774252939
979629944
1917628840
615706376
1807041724
1423534695
349489597
1838903296
1745438996
1419934740
517520065
539039049
540955165
694980843
1943871559
1296089656
1440184891
1042305834
1287392305
364073419
425032203
1021649991
724008987
243415601
413154583
1521471124
95646573
1346133737
403768016
1518584895
851017663
1263279673
16207477
675308439
76577354
1049196397
1578961337
2001879994
643538024
806267406
1362244042
774718420
356720720
1496889818
888393516
976499771
1258706767
92685381
105247165
695013360
1446669664
507253994
329629667
48249059
883788187
1060463956
1986837104
1208940156
1316237024
2089247012
1377447734
301827889
381244703
1376476441
1405446050
988896485
2117722298
1102496535
608347832
731178718
1038773901
1238159983
826729292
780922574
1212215971
969627131
364514215
1544051165
1189475434
1362284268
2041478887
939514706
1435876482
1671948512
511674098
1969007497
1928459157
44604558
395584797
1257092200
533197974
613199291
2073376887
1331988101
428429616
1874380060
969854779
652735638
1521017500
460661203
1581162921
409144951
991983471
1741735590
1955496149
1355106087
385832362
1483181855
2088338222
116655218
1683694247
249000095
1083706594
1552995217
393634341
2108903216
1204373813
1267147284
1233513637
2003881997
1280331806
320307619
2062883154
2018613047
1960885260
300805115
1413017354
1378782977
1807617973
472391234
1623437663
331686463
1401667807
1960097260
347182498
802959121
1423864751
551406037
1279923920
1696299388
489361608
2089060034
2063160806
1963373053
402763251
1846555911
1680125906
925833835
676684521
574672234
1889853977
425631865
707047659
1356150440
1613689342
1540054589
1204034901
1604831627
1549916835
1956838780
931289147
1886479795
1771797485
1941209701
1892390528
1340117311
929347993
143643878
1999302999
881385479
1000973501
1341531315
1827231667
187822081
1830786344
776710762
8396713
1003252412
612272697
2039171409
936906920
1196611890
1173979326
2056905650
1584581654
71591544
2072566499
635651156
2074584686
1564182078
204470296
1443913513
1154905670
1074549032
1453865697
421743253
1718272933
1771633046
684007650
286177352
158095248
1297241884
869427602
897222186
211858401
46741397
694921756
2002963912
1460618331
539677991
1402924566
2068110739
1696154860
1207844935
436627615
588873822
1348709131
1974775505
435719724
324490662
1162113204
1448586304
2117706875
1972538550
902341014
1727103441
1942023604
447085969
1834163107
2108201963
382980382
147714100
201474990
343145708
1142491645
912040689
1661339973
296967155
345160240
1065871831
1063801122
321111384
951773580
844816790
1666436748
1987725830
891194530
637616768
2125960805
2032006654
1136475186
1628811209
602586586
1033495112
1719906913
308630103
2022536916
654310205
1966152707
825051454
1569636728
2107388782
1516881811
969314942
536295337
1043823570
764527572
1927338695
1809295121
1072039867
511043868
338358270
1886607228
1677230721
590363268
793460894
2060555829
503534476
1727856093
802540473
1949346334
1261981212
928171235
132432511
329992230
1933928964
108767761
930742753
1747764813
974454801
903657569
1940674750
1598908066
1137134666
1503121436
480294340
924712883
303536155
1062868330
167456770
1683325046
206923748
1835774370
458239235
1237430273
1502049212
180277856
1044936349
1342955661
1339842225
2061867420
642043413
199082868
366430943
1145149088
1508250167
1333328226
1080467692
35188701
186288338
1619445419
1467552285
752610299
452941609
219894040
165176052
1043097269
1835497342
1077083608
1266090631
763542482
295093717
1582564153
1659482946
1663496529
152781055
426879041
2146885597
2107178222
332820597
576299481
1553963609
2102194148
660633775
1783186827
1127863322
1372152733
190882719
2007175981
249071073
688605834
1858688984
936714990
1632226992
1480994948
560079742
363382013
719690805
2003520824
1121391815
2005934489
562593906
387224290
1231000025
1830629154
1261688971
280996815
204376613
1228587268
285604876
1432433060
986510548
847983135
1886796895
1625096747
1924509227
1149475823
651033351
884843002
1794490473
948598215
127886033
446068731
390075548
586834526
1128647897
15317050
1160043170
1973386187
807612277
1820742828
561547899
17003078
1053581876
1199883938
1965913949
1903479771
2027750011
1148178513
468475376
857408559
1937174924
1681204029
1813508447
1492076608
1859029204
2088032828
53369391
217076025
993640115
924414740
1305490730
426866460
322708698
1044718092
396581653
70571746
296720722
1077076489
303402037
1486092951
570678640
897990215
1062513787
546710051
667255610
604847580
347099748
1031021326
1466334167
2036104048
1076147179
1330027462
21347166
681424435
336371890
952129471
2083011827
1406813610
421949032
648153598
1907056337
1705646060
1044262583
1082911824
425326360
731426580
33981240
1301673954
1447414589
1966710888
1779373546
713630451
1291246722
1408204337
20572969
310170055
1214151077
1842614522
1693739990
1059138617
984558870
1028467686
1854079186
205755050
1318471152
925947043
1747020775
1630766447
990915753
327053373
2045412922
1397337148
2073328983
1725550281
1020891162
1463395688
1802315032
1317161149
79039514
1361639541
1753861511
1754237473
1217856674
976627247
1205932142
875170800
1872037014
1600584129
1390338229
728483016
1298019824
440597092
68811760
1235135177
1497659800
1568700927
1384956611
405775118
1744521217
1623128378
913669641
2076007913
431737724
176363467
993841341
906567851
1875160382
1595951805
2083960068
1985288501
1912018472
1222173722
1735373615
1412075668
1533493856
2096032251
1800899628
1954552653
1824189578
1066266407
65820381
914775701
343224328
2130106342
975275229
788735148
324190470
1174263162
470441287
156004451
1657743939
526030215
2041633089
616025922
2021335625
1203034819
472280493
1948248736
796192344
1036376860
792278385
1320341150
1360224916
2134486523
347337821
273206863
1774459075
236197761
470446444
2081628756
1321110914
291725866
313423441
269992705
530350855
1351670962
1207715134
1300016353
26465038
81375940
963600721
1832546882
156538065
2132725701
1993925289
1702561235
153752644
2142449594
2132159602
253873178
548158196
676169733
51808496
890477302
498260361
643076413
199347447
796398286
1073782291
1999810835
1732244413
766346672
2062460787
1202702479
496361755
124685436
616762407
401078548
1705269401
534580997
507456283
1137154825
2095913626
142325072
2103739253
591107308
935983283
745001992
1886219897
1590543239
1710421531
303514790
551659618
1270223318
698789828
226204728
2051991672
1603324909
1341498153
132819586
1429535945
642124441
1466313535
1618049047
110830689
440522266
2088020974
1187818161
767004323
1451858946
151068860
1826978878
1707298583
13476046
252670647
1942832311
58729348
10271838
1608808941
434805386
349965440
2120932356
1850664459
698885260
290975673
1272981817
1783708926
1104205435
276648000
1222561462
909945179
1202408902
1828578365
1047290715
454983663
1453285211
704814986
416144229
214459314
602027749
1244066480
1591136947
436576242
1374291205
1127179100
450821372
1157216597
1235922767
573018467
1291872191
529446859
109864375
531863141
1412660829
865622487
1167559233
1470324823
2098330694
802254807
604235856
727123183
1827520037
1009099477
1371798642
1663415081
975657610
327833556
1162026454
853647799
1308012798
1695487758
1898698894
2090500579
1058655639
1620278613
2116251438
265053968
563713269
1109935976
1387594278
2088803119
1753800401
1266859378
796204158
864252144
1458939171
923099008
1855649446
381969049
870057008
1646650300
1836863940
970505466
1212509195
1922349932
1462860868
1481220124
2019024539
175084937
2141533499
965075547
613091198
1518723543
959680405
1481498532
811578587
1254122828
1218934668
1309117879
737345032
439366400
1955947429
1395829122
2028197615
1802027308
637393390
54531070
1491290343
1587439628
1722249820
475943077
676721825
1022137292
39159053
2132149911
438569082
1120209293
1988482463
385417043
1328772965
1540269773
811211353
1723020846
1885743633
2108272849
680872728
1658663948
648311435
455784477
2106273754
324909639
256437608
456685491
147318344
1289954049
196694242
62747261
1029139860
1309050110
1455069844
1701712158
1686936364
348546969
980418794
1464740771
1766889441
954096001
1593013352
591374515
333566400
1086243458
1719619288
1772331885
734839224
1282488429
829142399
1816597690
454663728
1185434090
1080033210
32804646
858428999
1465532719
307604645
40873314
2058181187
1770624086
826566718
422655236
765017221
386958730
1133085007
1887125818
1026390658
856733910
625799554
145811530
595018777
2088935636
590633544
1430757656
1567163396
776106838
170621459
649826841
209329636
112325335
1108890055
766971150
2032890195
293325790
1928375995
120547270
1532566679
214837990
1110866415
470981363
2083146070
615484394
1057609594
759500105
1679896964
785081306
1703716417
241962228
1907314168
952912615
1401946678
777161685
392846189
142898450
1281566881
2000180095
248255511
539277876
934587269
1595533271
1754885035
1748648463
1545490015
238367726
99909668
267340580
1855052034
1090538170
931733026
68918419
1487135462
745434843
1759752099
879983550
1230635364
1452828500
180017358
1323851603
1874408897
45399684
1954554272
415886634
2034297479
1887765794
543525904
2058641013
1157471230
1416851973
1080140378
1436333067
1316077761
1064780070
2136740176
260989699
432677895
790524406
967608385
689243460
1085695195
58706692
1244922281
400316585
566829608
178782708
536631925
310673864
217568631
197412210
604517964
869011172
840825381
1152400803
1602457972
208303229
249021029
1202923407
1053903158
1594045239
586171814
2049537059
1809041677
1568844988
911972527
1347943490
1579769390
1737626157
527475812
1492841121
1789879943
381985463
892073578
1595100897
1788873972
275370510
833822899
397361570
1767645861
825515587
1402000246
787913231
346396695
1286461448
153169907
1019036053
1649121767
1909232377
1479389387
949907998
917971276
791094414
1067577528
132703643
354546501
2107058586
34663342
662842239
978512704
1281954648
369069914
1906949403
1787202778
197541270
2233223
1157598031
501485315
354639905
787391878
763673734
4421769
1621049749
422041789
255186740
1042019934
2090007932
724160471
366435196
8168751
1743157010
1843393891
2025967083
1845609328
451007368
870847231
1168328693
1955329238
165042819
489864331
1836875197
1905244209
546254756
188216442
1421801619
1804858695
1696952967
1910269031
1207325259
987012220
1468919661
1373732753
921591918
80261714
1665390731
808031345
1599838995
1018354538
740664253
1277058010
903101442
2026891849
1579311235
486047840
803835111
2113905570
1149279800
444745301
488355440
1739602764
590491886
1924414514
2016328327
201373710
1013727125
528126277
1468815081
295963832
2082649087
515100636
1400386169
1648020105
868324460
665680600
876534627
495318762
1953053726
1231800987
45493158
1471791852
586821397
1132584908
64375992
1932609519
67263408
594463131
1085533448
1815964721
571956740
411442768
1296704620
1998596979
1590759609
1995641583
776139483
1073308170
1120620809
871715190
1020551271
544144912
2130938984
111772514
1556764744
1802064904
2142115520
418962090
491807799
233257012
1915371281
1773874476
1384261929
418153835
91114532
1173483554
2125685200
2130125846
313239938
992431843
580023685
2058176972
926112536
1048160989
1279363021
591424286
613792987
1786347863
151501740
1424159538
2023642785
72422526
1668107864
1921206877
515422850
1538401481
944380787
418161593
206436689
2035913114
1354574269
1548195761
264202027
1418082623
129321303
422518234
1740858890
810032507
768800466
646989369
796493833
1218544506
1260714506
2050985673
1891693823
612520706
1144035754
1432770552
1059439387
1852139416
1391932526
831687533
183295352
1585562690
109017657
1938368885
2087921113
951333726
1509669852
2039013755
1832752936
124131832
1536642306
643493804
925987245
817150943
773556124
1318258043
562016089
1872672391
621620673
677651830
1864183774
1993532975
488738249
1389225143
2063884032
123225285
822810817
481572261
1891611934
715134767
45226164
652847664
1158049481
1362093379
166643561
1180922679
1893834014
447012629
2032870100
1940280008
1731459485
835763307
2056678515
525083012
229281329
1687998999
1095027473
365986921
587064745
997377593
848383686
1130153975
349199011
561092558
1941803247
558420082
830916644
651731467
1198453474
1375433895
1088116966
338135296
1593766434
1333215147
1282410307
552000786
1615448786
1766640220
1104603879
171534502
950599695
1777193009
590947583
124157713
2084992101
2011288842
1905152700
501574510
415200477
822373090
599299016
1082374632
1789848087
752980824
332759524
598531109
699049153
248090255
588048841
407509514
669879800
191278277
1452284390
781930130
666285785
6574507
1893257503
901555411
2042200882
814546508
1576117252
1387476869
2133917011
121558999
1372850801
923980145
1544021795
1996502247
782264844
1473702751
520661352
516248524
1859489210
733286739
1011306241
1120818346
1050809252
208262227
1997683619
1570187490
1017813332
1931299599
167549934
286379782
586845721
574247688
525618302
254271646
1122516185
89807561
1624967980
422382125
1248354617
705907551
170472709
1304888946
2011212204
1215486097
1899954097
1274007235
728727720
1161064628
1906849220
1173426713
1316944864
638670037
95744430
1691508378
551702919
1852805825
458822899
1634569156
1804689407
974896628
74981608
1206902317
1516228165
482802119
883368847
2141789795
1849407989
659732531
378820419
1615428795
185692845
1900544242
1100814293
1022115469
1423919390
11191250
2097975246
1919848459
526896463
606512700
1200184115
524518955
1219780356
786081472
211196283
976114117
186847735
954577091
1444460562
1206661433
1900159870
1912329584
1119022043
1755168846
1438502487
1416309015
1467001600
1271244655
1755586634
236025427
1258998686
26238141
803608021
1158911998
1676426305
226011021
300718229
560207668
1689432684
