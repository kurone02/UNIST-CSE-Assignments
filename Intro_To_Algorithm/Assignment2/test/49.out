1504598270573